andela/wg-fortem
# 166152338 Adds hover over body part to get exercises

#### Title

A user should be able to hover a body part to obtain a list of exercises

#### Description



When a user hovers over a body part, they should be able to get exercises for that body part. This creates a great and memorable user experience

#### Type of change








#### How Can This Be Tested?
To be able to test the feature, check out the branch `story/166152338-hover-body-get-exercises` and then `python manage.py runserver`.
Navigate to the URL `http://127.0.0.1:8000/en/exercise/muscle/overview/` on your browser
Hover over the image. 
<img width="1440" alt="Screenshot 2019-06-11 at 20 18 35" src="https://user-images.githubusercontent.com/40719885/59292636-21acf480-8c86-11e9-9493-a0b26b0f1be7.png">
#### Acceptance Criteria: 



#### Checklist:
- [x] Adds svg for front muscles
- [x] Adds svg for back muscles
- [x] Adds CSS styling to allow hover

#### PT stories
[#166152338](https://www.pivotaltracker.com/story/show/166152338)
